src/cascadia/PublicTerminalCore/HwndTerminalAutomationPeer.cpp

```diff
@@ -153,7 +153,7 @@ void HwndTerminalAutomationPeer::NotifyNewOutput(std::wstring_view newOutput)
         return;
     }
 
-    const auto sanitizedBstr = SysAllocString(sanitized.c_str());
-    static auto activityId = SysAllocString(L"TerminalTextOutput");
-    LOG_IF_FAILED(UiaRaiseNotificationEvent(this, NotificationKind_ActionCompleted, NotificationProcessing_All, sanitizedBstr, activityId));
+    const auto sanitizedBstr = wil::unique_bstr(sanitized.c_str());
+    static auto activityId = wil::unique_bstr(L"TerminalTextOutput");

```

Comment:
ah, this is a bit of a weirdness.

I don't know if this is a legal way to create a unique_bstr.

Try using `wil::make_bstr_nothrow()` instead!